packages/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.1.0
```

Comment:
Are breaking changes supposed to be major version numbers (i.e., 4.0 in this case)?